src/features/commands.ts

```diff
@@ -567,6 +567,40 @@ Here's an article explaining the difference between the two: https://goshakkk.na
       });
     },
   },
+  {
+    words: [`!jwt`],
+    help: `Describes reasoning for and against the use of JWT tokens againt using standard sessions`,
+    category: "Reactiflux",
+    handleMessage: (msg) => {
+      msg.channel.send({
+        embeds: [
+          {
+            title:
+              "Is JWT the right approach for my applications authentication?",
+            type: "rich",
+            description: `
+            **TL;DR: JWT is widely overused as the go to authentication choice over sessions in most situations. Use this bot to form your opinion on wether it is a good fit for you.**
+            
+            A lot of developers starting their journey into programming will, at some point, hit the question of how to work with backend authentication.
+            They will undoubtably come across the multitude of JWT tutorials on the subject. But is this the correct approach?. Most of the time the answer will be no. 
+            So what is the alternative? Sessions. Sessions have been tried and tested for decades, with a lot of back end frameworks supporting them out of the box.
+            They take up miminal setup times, are completely managed by the server and cannot, if correctly set up , be accessed by the client
+            See below for some helpful links to help you decide which is the best solution for you:
```

Comment:
The TL;DR is written a bit aggressively ("widely overused" I think will be received as a hot take, and "use this bot to form your opinion" seems optimistic), and I think the "question talking" form of a lot of the rest could be pared down into simpler sentences. What do you think of this?

```suggestion
            description: `Most of the time, JWTs aren't the best approach for working with backend authentication, despite the multitude of tutorials that use JWT. Session have been used for decades, with a lot of back end frameworks supporting them out of the box.

See below to help you decide which works best for you:
```

Maybe it makes sense to add 1 more sentence briefly summarizing why sessions are a better default than JWTs? Here's how they look rendered, before:
<img width="539" alt="Screen Shot 2022-09-23 at 11 35 47 AM" src="https://user-images.githubusercontent.com/1551487/191999066-f1419887-3a9f-43c0-a74c-d9c6b16b1cc7.png">

after:
<img width="538" alt="Screen Shot 2022-09-23 at 11 35 52 AM" src="https://user-images.githubusercontent.com/1551487/191999070-cb15d0ea-335d-4006-b62f-5588a9387de8.png">

Reply:
This seems like an improvement. Though I wonder, aren't there some use cases where JWT are actually preferred? This reminds me of popular advise against MongoDB and other document stores, which are bad for most data, but actually very good in some edge cases like analytics data.

Reply:
thanks for the feedback. have incorporated this into the latest push